### PR TITLE
Rajan/model file system loader

### PR DESCRIPTION
**Pull request by rajanjedi**
Load models from local file system.

### CODE REVIEW

File: rlclientlib/model_mgmt/file_model_loader.cc [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+      if (!in_strm.good()) {
+        RETURN_ERROR_LS(_trace, status, file_open_error) << " file_name = " << _file_name;
+      }
+    }
+    return error_code::success;
+  }
+
+  int file_model_loader::get_data(model_data& data, api_status* status) {
+
+    std::ifstream in_strm(_file_name.c_str(), std::ios::in|std::ios::binary|std::ios::ate);
```
Comment (jackgerrits) [inline rlclientlib/model_mgmt/file_model_loader.cc]: How costly is opening the file? Perhaps we should check if it was modified prior to opening it?

Comment (ataymano) [reply rlclientlib/model_mgmt/file_model_loader.cc]: it is done on higher layer - see get_file_modified_time above,   which is called before

Comment (jackgerrits) [reply rlclientlib/model_mgmt/file_model_loader.cc]: But in the sequence of the get_data function, it is called after creating the ifstream object though?

Comment (ataymano) [reply rlclientlib/model_mgmt/file_model_loader.cc]: oh yes you right

**Comment by yannstad** [comment]
Can this be merged?

File: rlclientlib/model_mgmt/file_model_loader.cc (hunk 1 above)
Comment (rajan-chari) [reply rlclientlib/model_mgmt/file_model_loader.cc]: Does not happen in the hot path.   Not clear what the perf impact is.

**Comment by rajan-chari** [comment]
@yannstad  Merged

**Pull request merged**